ui/src/containers/Header/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';  
```

Comment:
this line has 2 extra spaces in the end due to which it's showing linting error. Can you please remove that.